src/framework/ui/view/iconcodes.h

```diff
@@ -99,8 +99,8 @@ class IconCode
         ARROW_UP = 0xEF37,
         POSITION_ARROWS = 0xEF38,
         TEXT_ALIGN_BASELINE = 0xEF39,
-        TEXT_ALIGN_ABOVE = 0xEF3A,
-        TEXT_ALIGN_UNDER = 0xEF3B,
+        TEXT_ALIGN_UNDER = 0xEF3A,
+        TEXT_ALIGN_ABOVE = 0xEF3B,
```

Comment:
Strange, I always believed "UNDER" being the opposite/counterpart to "OVER", and "ABOVE" the opposite/counterpart of "BELOW"...

Reply:
I agree, Above is the opposite to Below. I'm not sure of the Under and Over though

Reply:
So renaming TEXT_ALIGN_UNDER to TEXT_ALIGN_BELOW seems a good option. Ot even keeps the legnth, so doesn't srew up score formatting

Reply:
Why not just `TEXT_ALIGN_TOP` and `TEXT_ALIGN_BOTTOM`?

Reply:
@cbjeukendrup Yeah I agree with you. Excel and Google Sheets uses them same terms

Reply:
Works too, and probably even better as it matches other similar defines

Reply:
Thanks, I was about to make the same suggestion (i.e use "Top" and "Bottom" instead of "Above" and "Below"). When the reference point is the same, text that uses the **top** alignment option will actually appear _below_ text that uses the **bottom** alignment option.

![image](https://user-images.githubusercontent.com/11011881/186447332-112e690b-b5f0-4c75-903e-81e2f6235c8d.png)

Similarly, **align left** will put text on the _right_ of the reference point, so we should be consistent with terminology.

Reply:
@shoogle Nice! I was also thinking of adding tooltips to those buttons. There aren't any now. What do you think?

Reply:
@HemantAntony, sure! I was thinking the same and was about to create an issue. But let's merge this and put the tooltips in a separate PR.

Reply:
Okay cool!